scripts/cursor_pr_automation.sh

```
@@ -0,0 +1,243 @@
+  if [[ -z ${token} ]]; then
+    err "No GitHub token available for API fallback. Set GITHUB_TOKEN or authenticate gh."
+    return 1
+  fi
+
+  api_url="https://api.github.com/repos/${owner}/${repo}/pulls"
+  payload=$(cat <<JSON
+{
+  "title": "${TITLE//"/\"}",
+  "head": "${branch//"/\"}",
+  "base": "${BASE_BRANCH//"/\"}",
+  "body": "${BODY//"/\"}"
```

Shellcheck (0.11.0)

[error] Couldn't parse this double quoted string. Fix to allow more checks.

(SC1073)